test/parallel/test-fs-read.js

```diff
@@ -2,21 +2,24 @@
 const common = require('../common');
 const assert = require('assert');
 const path = require('path');
+const Buffer = require('buffer').Buffer;
 const fs = require('fs');
 const filepath = path.join(common.fixturesDir, 'x.txt');
 const fd = fs.openSync(filepath, 'r');
 const expected = 'xyz\n';
+const bufferAsync = Buffer.allocUnsafe(expected.length);
+const bufferSync = Buffer.allocUnsafe(expected.length);
 
 fs.read(fd,
+        bufferAsync,
+        0,
         expected.length,
         0,
-        'utf-8',
-        common.mustCall((err, str, bytesRead) => {
-          assert.ifError(err);
-          assert.strictEqual(str, expected);
-          assert.strictEqual(bytesRead, expected.length);
+        common.mustCall(function(err, bytesRead) {
```

Comment:
We will be getting the buffer object as the third argument. You might want to check if it is the original `bufferAsync` object itself.